packages/kit/src/runtime/app/server/index.js

```diff
@@ -27,7 +27,7 @@ export function read(asset) {
 	}
 
 	// handle inline assets internally
-	const match = /^data:([^;,]+)?(;base64)?,/.exec(asset);
+	const match = /^data:([^;,*?)?(;base64)?,/.exec(asset);
```

Comment:
This is an invalid regex. And anyway, `/^data:([^;,]+)?(;base64)?,/` will already match `'data:,Hello%2C%20World%21'`.